### ISSUE TITLE
pageby_header = FALSE does not hide page header for all pages

### ISSUE THREAD

**Issue by tdwils** [Merck/r2rtf#80]
When we set pageby_header = FALSE, the page header is not hidden. It is displayed at the beginning of each new page_by variable, and then hidden on subsequent pages.

For example, we want to create a table with one subject per page, but we don’t want to show the page header.

Here is code for a simple table:

```
library(r2rtf)
library(dplyr)


data(r2rtf_adae)

ae_t1 <- r2rtf_adae %>%
  filter(USUBJID %in% c("01-701-1383", "01-701-1444")) %>%
  mutate(SUBLINEBY = TRTA) %>% 
  mutate(PAGEBY = USUBJID) %>%    
  select(USUBJID, ASTDY, AEDECOD, SUBLINEBY, PAGEBY) 

ae_tbl <- ae_t1 %>%
  arrange(SUBLINEBY, USUBJID, ASTDY) %>%
  rtf_page(orientation = "landscape", nrow = 15) %>%
  rtf_page_header() %>%
  rtf_title("AE Listing") %>%
  rtf_body( 
    subline_by = "SUBLINEBY",
    page_by = "PAGEBY",
    group_by = c("USUBJID", "ASTDY"),
    new_page = TRUE,
    pageby_header = FALSE) 

ae_tbl %>%
  rtf_encode() %>%
  write_rtf("rtf/pageby-ae-listing.rtf")
```

The pageby_header argument is set to FALSE, but the page header still displays on pages 1 and 3, where a new subject ID starts. The page header is hidden on page 2, which is a continuation of the subject ID (01-701-1383) from page 1.

[pageby-ae-listing_actual.pdf](https://github.com/Merck/r2rtf/files/7730049/pageby-ae-listing_actual.pdf)

We would like the page header to be hidden for all pages, like this: 

[pageby-ae-listing_desired.pdf](https://github.com/Merck/r2rtf/files/7730051/pageby-ae-listing_desired.pdf)

Is there a way we can do that? Thank you!

**Comment by elong0527** [comment]
The intention of `pageby_header = FALSE` is to control the header to display only once. PR #81 clarified the information.

With current `r2rtf` version, the desired layout needs to be handled by multiple files. 

1. create one AE listing file per subject. 
2. combine them into one RTF file following the steps below.

```
library(r2rtf)
library(dplyr)


data(r2rtf_adae)

ae_t1 <- r2rtf_adae %>%
  filter(USUBJID %in% c("01-701-1383", "01-701-1444")) %>%
  mutate(SUBLINEBY = TRTA) %>% 
  mutate(PAGEBY = USUBJID) %>%    
  select(USUBJID, ASTDY, AEDECOD, SUBLINEBY) 

x <- split(ae_t1, ae_t1$USUBJID)

x_encode <- lapply(x, function(tbl){
  
  tbl %>%
    arrange(SUBLINEBY, USUBJID, ASTDY) %>%
    rtf_page(orientation = "landscape", nrow = 15) %>%
    rtf_page_header() %>%
    rtf_title("AE Listing") %>%
    rtf_body( 
      subline_by = "SUBLINEBY",
      group_by = c("USUBJID", "ASTDY")) %>% 
    rtf_encode()
  
})

# Combine other RTF body to the first one  
x_encode[[1]]$body <- paste(lapply(x_encode, function(x) x$body), collapse = r2rtf:::as_rtf_new_page())

x_encode[[1]] %>% write_rtf("output.rtf")
```


If you do not have to display each subject in a new page, I assume you already know you can do it as below. 

```
library(r2rtf)
library(dplyr)


data(r2rtf_adae)

ae_t1 <- r2rtf_adae %>%
  filter(USUBJID %in% c("01-701-1383", "01-701-1444")) %>%
  mutate(SUBLINEBY = TRTA) %>% 
  mutate(PAGEBY = USUBJID) %>%    
  select(USUBJID, ASTDY, AEDECOD, SUBLINEBY) 

ae_tbl <- ae_t1 %>%
  arrange(SUBLINEBY, USUBJID, ASTDY) %>%
  rtf_page(orientation = "landscape", nrow = 15) %>%
  rtf_page_header() %>%
  rtf_title("AE Listing") %>%
  rtf_body( 
    subline_by = "SUBLINEBY",
    group_by = c("USUBJID", "ASTDY")) 

ae_tbl %>%
  rtf_encode() %>%
  write_rtf("ae-listing.rtf")
```

**Comment by tdwils** [comment]
That works. Thank you for the solution and the code. I was able to reproduce it.  

Thanks,
Teresa

**Comment by GlicketySplit** [comment]
For the latter ("If you do not have to display each subject in a new page, ..."), what if USUBJID is a long string but I still want to display this on a single row without top/bottom borders? 

[similar to SUBJLINE used here](https://merck.github.io/r2rtf/articles/example-sublineby-pageby-groupby.html)

**Comment by elong0527** [comment]
Please create a new issue and provide example code and output of what you want to achieve.
